exporter/containerimage/exptypes/attestations.go

```diff
@@ -0,0 +1,16 @@
+package exptypes
+
+const (
+	MediaTypeDockerSchema2AttestationType = "application/vnd.in-toto+json"
```

Comment:
Might want to sync with the containerd people on what to call this - since we'll need to get this to be recognized, or we'll see the the `"reference for unknown type: application/vnd.in-toto+json"` warning from https://github.com/jedevc/buildkit/blob/wip-attestations-sbom/vendor/github.com/containerd/containerd/remotes/handlers.go#L83

In the meantime, not quite sure where these media type definitions belong...